en/lessons/specifics/plug.md

```diff
@@ -80,7 +80,7 @@ It receives a `%Plug.Conn{}` connection struct as its first argument and is expe
 
 We need to tell our application to start up and supervise the Cowboy web server when the app starts up.
 
-We'll do so with the [`Plug.Cowboy.child_spec/1`](https://hexdocs.pm/plug_cowboy/Plug.Cowboy.html#child_spec/1) function.
+We'll do so with the [`Plug.Adapters.Cowboy.child_spec/1`](https://hexdocs.pm/plug_cowboy/Plug.Cowboy.html#child_spec/1) function.
```

Comment:
So this actually should be `Plug.Cowboy.child_spec/1`. Latest version of Cowboy (Cowboy2) [instructs users that `Plug.Adapters.Cowboy` is deprecated.](https://hexdocs.pm/plug/Plug.Adapters.Cowboy.html)

Reply:
Thanks so much! Not sure how I missed that 🤦‍♂️